app/code/Magento/MediaGalleryUiApi/Api/Data/InsertImageDataInterface.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\MediaGalleryUiApi\Api\Data;
```

Comment:
Let's move this interface to the suggested namespace
```suggestion
namespace Magento\MediaGalleryUi\Model;
```